vowel_count_code.py

```diff
@@ -1,4 +1,7 @@
-import re
-
 def vowel_count(string):
-    return len(re.findall(r'[aeiouAEIOU]', string))
\ No newline at end of file
+    vowels = "aeiouAEIOU"
```

Comment:
What does Python do when you say ` variable in string`?   It doesn't know if it's in there or not, so it has to iterate over the whole thing to check each position to see if it matches.

Change vowels [to a Set](https://docs.python.org/3.10/library/stdtypes.html#set-types-set-frozenset).   Sets are hashed, so the inclusion check is an [O(1)](https://en.wikipedia.org/wiki/Big_O_notation) lookup.